modules/ROOT/pages/users.adoc

```diff
@@ -69,64 +69,59 @@ include::partial$include-nav-users.adoc[]
 [[grant-user-permissions]]
 == Grant Permissions to Users
 
-Organization administrators manage user access using xref:teams.adoc[teams] or xref:roles.adoc[roles] (deprecated), or by granting permissions to users individually. Teams and roles functionality are mutually exclusive.
+Organization administrators manage user access using xref:teams.adoc[teams], or by granting permissions to users individually. 
 
-In the *Users* page, click a user name to access more information about that user, manage and view permissions, manage and view teams or roles, or reset their password.
+In the *Users* page, click a user name to access more information about that user, manage and view permissions, manage and view teams, or reset their password.
 
 By default, in every new organization and business group, you can assign the following:
 
 * Assign API permissions: Select the name of the API you want to give access to, then pick a version and permission.
 * Assign Runtime Manager permissions: Select the name of the Runtime Manager environment to give access to, then pick a permission.
 * Assign teams: Select the name of the team to give access to.
-* Assign roles (deprecated): Select the name of the role to grant. Check the xref:roles.adoc[roles] section for a description of the default roles within an organization and business group.
```

Comment:
Obviously we need to get rid of any roles or legacy UI documentation that is irrelevant, but do we still want to keep documentation for roles functionality that still exists, even if deprecated? For example, the role assignment functionality in this bullet point is still used by some customers, even though it's not the preferred approach anymore.